File: app/src/main/java/com/bbi/bizbulls/model/PersonalReferenceViewRes.java
package com.bbi.bizbulls.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class PersonalReferenceViewRes {

    @SerializedName("data")
    @Expose
    private List<Data> data = null;

    public List<Data> getData() {
        return data;
    }

    public void setData(List<Data> data) {
        this.data = data;
    }
    public class Data {

        @SerializedName("id")
        @Expose
        private String id;
        @SerializedName("fullname")
        @Expose
        private String fullname;
        @SerializedName("relation")
        @Expose
        private String relation;
        @SerializedName("sex")
        @Expose
        private String sex;
        @SerializedName("age")
        @Expose
        private String age;
        @SerializedName("occupation")
        @Expose
        private String occupation;
        @SerializedName("location")
        @Expose
        private String location;
        @SerializedName("contact_number")
        @Expose
        private String contactNumber;
        @SerializedName("address")
        @Expose
        private String address;

        public String getId() {
            return id;
        }

        public void setId(String id) {
            this.id = id;
        }

        public String getFullname() {
            return fullname;
        }

        public void setFullname(String fullname) {
            this.fullname = fullname;
        }

        public String getRelation() {
            return relation;
        }

        public void setRelation(String relation) {
            this.relation = relation;
        }

        public String getSex() {
            return sex;
        }

        public void setSex(String sex) {
            this.sex = sex;
        }

        public String getAge() {
            return age;
        }

        public void setAge(String age) {
            this.age = age;
        }

        public String getOccupation() {
            return occupation;
        }

        public void setOccupation(String occupation) {
            this.occupation = occupation;
        }

        public String getLocation() {
            return location;
        }

        public void setLocation(String location) {
            this.location = location;
        }

        public String getContactNumber() {
            return contactNumber;
        }

        public void setContactNumber(String contactNumber) {
            this.contactNumber = contactNumber;
        }

        public String getAddress() {
            return address;
        }

        public void setAddress(String address) {
            this.address = address;
        }

    }
}

File: app/src/main/java/com/biz/bizbulls/model/PersonalReferenceViewRes.kt
package com.biz.bizbulls.model

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose

class PersonalReferenceViewRes {
    @SerializedName("data")
    @Expose
    var data: List<Data>? = null

    inner class Data {
        @SerializedName("id")
        @Expose
        var id: String? = null

        @SerializedName("fullname")
        @Expose
        var fullname: String? = null

        @SerializedName("relation")
        @Expose
        var relation: String? = null

        @SerializedName("sex")
        @Expose
        var sex: String? = null

        @SerializedName("age")
        @Expose
        var age: String? = null

        @SerializedName("occupation")
        @Expose
        var occupation: String? = null

        @SerializedName("location")
        @Expose
        var location: String? = null

        @SerializedName("contact_number")
        @Expose
        var contactNumber: String? = null

        @SerializedName("address")
        @Expose
        var address: String? = null
    }
}